CHANGELOG.rst

```diff
@@ -5,6 +5,103 @@ Dellemc.Enterprise_Sonic Release Notes
 .. contents:: Topics
 
 
+v2.3.0
+======
+
+Release Summary
+---------------
+
+| Release Date: 2024-0103
+| This release provides the functionality enhancements listed below, along with fixes for
+| problems found in regression testing or reported by users. The main functionality enhancements
+| provided are the following items.
+| 1) Complete the support for "replaced" and "overridden" state handling for all resource modules.
+| With this release, the required support has been added for any resource modules that were not
+| provided with this support for the 2.1.0 release.
+| 2) Provide initial support for the "--check" and "--diff" mode options for playbook execution. This
+| release provides the common utility support for these options for use by all resource modules.
+| It also provides the specific resource module changes required for implementation of the
+| functionality in many of the existing resource modules. (The "--check" and "--diff" mode support
+| for the remaining resource modules is planned for inclusion in the next release.)
+| 3) New resource modules for "Public Key Infrastructure", STP, and DHCP Snooping.
+| 4) Support for "ranges" of vlans (e.g '2-100') in tasks for the l2_interfaces and mclag resource
+| modules.
+| Please refer to the "CHANGELOG.rst" file at the top directory level of this repo for additional
+| details on the contents of this release.
+
+
+Minor Changes
+-------------
+
+- bgp - Fix the bgp CLI test base_cfg_path derivation of the bgp role_path by avoiding relative pathing from the possibly external playbook_dir (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/283).
```

Comment:
This PR updates only the regression tests.
Hence, the module could be updated as: 
```suggestion
- tests - Fix the bgp CLI test base_cfg_path derivation of the bgp role_path by avoiding relative pathing from the possibly external playbook_dir (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/283).
```